tests/end-to-end/ui/03-user-creation.js

```diff
@@ -22,7 +22,7 @@ describe('[User Creation]', function() {
 	it('it should create user', () => {
 		loginPage.gotToRegister();
 
-		loginPage.registerNewUser({username, email, password});
+		loginPage.registerNewUser({username, email, password, reason});
```

Comment:
Since the reason only shows up when the `approve users manually` setting is on this should be tested [here](https://github.com/RocketChat/Rocket.Chat/blob/develop/tests/end-to-end/ui/12-settings.js#L453)